### PR TITLE
Panel table fixes

### PR DESCRIPTION
**Pull request by pjfitzgibbons**
### Description
[Describe what this change achieves]

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by ps48** [comment]
@pjfitzgibbons Can you please fix the failing tests?

**Review by anirudha — APPROVED** [review]
Thanks

File: public/components/custom_panels/custom_panel_table.tsx [hunk 1]
```diff
@@ -133,19 +137,44 @@ export const CustomPanelTable = ({
...
 
-    dispatch(createPanel(newPanel));
+      const { id, ...newPanel } = {
+        ...sourcePanel,
+        title: newName,
+      };
+
+      dispatch(createPanel(newPanel));
+    } catch (err) {
+      console.log(err);
```
Comment (derek-ho) [inline public/components/custom_panels/custom_panel_table.tsx:158]: set error toast? I do not see toast code here?

**Review by derek-ho — APPROVED** [review]

File: public/components/custom_panels/custom_panel_table.tsx (hunk 1 above)
Comment (ps48) [reply public/components/custom_panels/custom_panel_table.tsx:158]: If toast is not ready, can we use `console.error`?

File: public/components/custom_panels/custom_panel_table.tsx [hunk 2]
```diff
@@ -193,7 +222,7 @@ export const CustomPanelTable = ({
         'Duplicate Dashboard',
         'Cancel',
         'Duplicate',
-        selectedCustomPanels[0].title + ' (copy)',
+        selectedCustomPanels[0].title + ' (copy)x',
```
Comment (ps48) [inline public/components/custom_panels/custom_panel_table.tsx:225]: I think this `x` is a typo. Should we remove this?

**Pull request merged**